Source/OCMockTests/OCMBoxedReturnValueProviderTests.m

```diff
@@ -53,12 +62,12 @@ - (void)testCorrectEqualityForCppProperty
 			"r^{GURL}";
 
 	OCMBoxedReturnValueProvider *boxed = [OCMBoxedReturnValueProvider new];
-	XCTAssertTrue([boxed isMethodReturnType:type1 compatibleWithValueType:type2]);
-	XCTAssertTrue([boxed isMethodReturnType:type1 compatibleWithValueType:type3]);
-	XCTAssertTrue([boxed isMethodReturnType:type2 compatibleWithValueType:type1]);
-	XCTAssertTrue([boxed isMethodReturnType:type2 compatibleWithValueType:type3]);
-	XCTAssertTrue([boxed isMethodReturnType:type3 compatibleWithValueType:type1]);
-	XCTAssertTrue([boxed isMethodReturnType:type3 compatibleWithValueType:type2]);
+	XCTAssertTrue([boxed isMethodReturnType:type1 compatibleWithValueType:type2 value:&value valueSize:valueSize]);
```

Comment:
I changed these tests to pass `NULL` and `0` for the last two arguments. Couldn't see a reason why not, and it saved creating the values here.